### DISCUSSION TITLE
Binding Information

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#13410] [Q&A]
How do I get that AvaloniaProperty is bind and get binding information like type (TemplateBinding, Binding, etc...) and related information (path, mode, etc...):

Avalonia Version: **11.x**

**Comment by timunie** [comment]
MyProperty.IsSet iirc

**Reply by workgroupengineering** [reply]
Thanks for the reply. IsSet only indicates whether ownership has been assigned. 

Example

```xaml
   <TextBlock Text="{Binding Message}" x:Name="test"/>
```

```c#
var info = BindingOpergations.GetBinding(test, TextBlock.TextProperty);

if(info is Binding bind)
{
   var path=bind.Path
}


```
